### DISCUSSION TITLE
Show CVSS score in table output

### DISCUSSION THREAD

**Discussion by hangrymuppet** [aquasecurity/trivy#7119] [Q&A]
### Question

Is there a way to show the CVSS score (if available) as an additional column to the default table output of an image scan? I am using the command below.

example command: `trivy image bitnami/redis:7.2.5-debian-12-r2`

### Target

Container Image

### Scanner

Vulnerability

### Output Format

Table

### Mode

Standalone

### Operating System

macOS

### Version

```bash
❯ trivy --version
Version: 0.53.0
Vulnerability DB:
  Version: 2
  UpdatedAt: 2024-07-08 18:11:40.452212609 +0000 UTC
  NextUpdate: 2024-07-09 00:11:40.452212459 +0000 UTC
  DownloadedAt: 2024-07-08 18:51:35.611608 +0000 UTC
Java DB:
  Version: 1
  UpdatedAt: 2024-06-18 01:05:26.456680153 +0000 UTC
  NextUpdate: 2024-06-21 01:05:26.456679993 +0000 UTC
  DownloadedAt: 2024-06-18 21:08:44.204563 +0000 UTC
```

**Comment by huornlmj** [comment]
+1 this is vital.

**Comment by itaysk** [comment]
the info is available in the json output so you can create your own formatter to display the results however you like. some poeple wrote and shared "output plugins" for trivy that does that.

**Comment by huornlmj** [comment]
@itaysk thanks, I know about the template option but my +1 here is to request to include it in the default table output.  Without the score it seems unnecessarily limited.
